spec/takeaway_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:menu) { {burger: 5.99, fries: 3.99, pizza: 7.99} }
+  let(:takeaway ) { Takeaway.new(menu) }
+
+  it 'shows the food menu with prices' do
+    takeaway
```

Comment:
Can switch takeaway to 'subject' instead. Means no amendments will be required if the name of the Class ever changes. Not too much of an issue, just an observation